en/FindDuplicates.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Find duplicates
+---
+
+# Find duplicates
+
+JabRef can look for duplicated entries inside a database.
+
+This feature is accessible directly through  **Quality -&gt; Find duplicates**.
+It is also used when [importing new entries](ImportInspectionDialog)
+from a supported reference format or directly from the Internet.
+
+Detection of potential duplicates is done by a distance algorithm.
+Extra weighting is put on the fields *author*, *editor*, *title* and *journal*.
+
+## Parallel display of potential duplicates
+
+![Screenshot of the parallel display](./images/FindDuplicatesWindow-ParallelDisplay.png)
+
+When two potential duplicates are found, their fields are displayed side-by-side.
+
+The differences between the two entries can be emphasized through the drop-down menu located
+at the upper right-hand corner of the window.
+Five ways of displaying the differences are offered:
+- **plain text**: as is, no emphasis
+- **show diff** - word: differences are shown in the right entry. Full words are struck out in red or underlined in blue.
+- **show diff** - character: differences are shown in the right entry. Individual characters are struck out in red or underlined in blue.
+- **show symmetric diff** - word: differences are shown on both sides. Words are underlined and displayed in color.
```

Comment:
I do not remember the logic, but will try to check it up. There is something different that should be explainable.